lists/my.csv

```diff
@@ -293,3 +293,58 @@ http://www.ytl.com.my,VOIP,VOIP,2014-04-15,citizenlab,
+http://www.facebook.com/boikotutusan,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.facebook.com/pages/barisan-nasional/219708793830,POLR,Political reform,2016-04-18,citizenlab,
+http://youtu.be/edaqtgy-azq,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/jkkfc,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.youtube.com/watch?&v=ui7xb3d2rna,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.youtube.com/watch?v=_1rrnggnm6a,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://pru13.info,POLR,Political reform,2016-04-18,citizenlab,
+https://www.facebook.com/supportprdap,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://www.facebook.com/Bersih4.0Malaysia,POLR,Political reform,2016-04-18,citizenlab,
+https://en.wikipedia.org/wiki/Bersih_4_rally,POLR,Political reform,2016-04-18,citizenlab,
+https://vulcanpost.com/353991/bersih-4-0-global/,POLR,Political reform,2016-04-18,citizenlab,
+https://vulcanpost.com/351121/10-items-to-bring-bersih-4-0/,POLR,Political reform,2016-04-18,citizenlab,
+http://www.straitstimes.com/asia/se-asia/what-you-need-to-know-about-malaysias-bersih-movement,POLR,Political reform,2016-04-18,citizenlab,
+http://www.therakyatpost.com/news/2015/08/25/bersih-4-0-gets-green-light-from-sarawak-police/,POLR,Political reform,2016-04-18,citizenlab,
+http://www.malaysiakini.com/news/306575,POLR,Political reform,2016-04-18,citizenlab,
+http://www.malaysiakini.com/news/310156,POLR,Political reform,2016-04-18,citizenlab,
+http://www.tallypress.com/fun/10-important-items-to-bring-to-bersih-4-rally,POLR,Political reform,2016-04-18,citizenlab,
+https://www.article19.org/resources.php/resource/38089/en/malaysia:-blocking-websites-to-prevent-protest-violates-international-law,HUMR,Human rights,2016-04-18,citizenlab,
+https://globalvoicesonline.org/2015/08/10/protests-as-malaysia-suspends-two-newspapers-following-a-corruption-expose/,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.sarawakreport.org/,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.theedgemarkets.com/my,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.wsj.com/articles/SB10130211234592774869404581083700187014570,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+http://www.bersih.org/,POLR,Political reform,2016-04-18,citizenlab,
+https://medium.com/@sarawakreport,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://medium.com/@sarawakreport/half-million-dollar-denial-najib-s-latest-1mdb-denial-is-directed-against-obama-s-key-aide-e9a46c71a210#.iwyok5h21,FEXP,Free expression and media freedom,2016-04-18,citizenlab,
+https://medium.com/@sarawakreport/najib-negotiates-his-exit-but-he-wants-safe-passage-and-all-the-money-exclusive-disclosure-8d7d44343b50#.rm2d681n3,FEXP,Free expression and media freedom,2016-04-18,citizenlab,

```

Comment:
If the URLs are in HTTPS it doesn't make sense to include the path in the URL. I would suggest having 1 in HTTPS and the others be in HTTP.